### PR TITLE
Pg mysql table selection

### PR DESCRIPTION
**Pull request by temandr**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
mysql table

**Comment by temandr** [comment]
@yonzhan Any updates on this PR?

**Comment by yonzhan** [comment]
@Juliehzl to help with this PR.

**Comment by temandr** [comment]
@yonzhan and @Juliehzl Any updates?

File: src/dms-preview/HISTORY.rst [hunk 1]
```diff
@@ -1,3 +1,9 @@
+0.12.0
+++++++++++++++++++
+
+* Allowing for table selection in PG migrations.
```
Comment (Juliehzl) [inline src/dms-preview/HISTORY.rst:4]: Is there a way for customer to know your change in help message？

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**